suse_migration_services/units/apparmor_migration.py

```diff
@@ -22,12 +22,31 @@
 # project
 from suse_migration_services.defaults import Defaults
 from suse_migration_services.logger import Logger
+from suse_migration_services.zypper import Zypper
 
 from suse_migration_services.exceptions import (
     DistMigrationAppArmorMigrationException
 )
 
 
+def install_patterns_base_selinux(root_path):
+    """Install patterns-base-selinux"""
+    Zypper.run([
+        '--no-cd',
+        '--no-cd',
+        '--non-interactive',
+        '--gpg-auto-import-keys',
+        '--root', root_path,
+        'in',
```

Comment:
Can we keep it explicit for better readability: `install` instead of `in` ?

Reply:
Fixed. I also update the command in `update_bootloader` as well; it's a bit out of scope for this PR, but since we are already refactoring them to use the Zypper class I think it's a good chance to update it together.